manifests/ols-rbac.yaml

```diff
@@ -30,4 +30,18 @@ rules:
 - apiGroups: ["authorization.k8s.io"]
   resources:
   - subjectaccessreviews
-  verbs: ["create"]
\ No newline at end of file
+  verbs: ["create"]
+---
+kind: RoleBinding
```

Comment:
Just curious, what is the significance of running components with `scc:anyuid`? Why can't we just leave it as before?

Reply:
It doesn't work without SCC

```text
pods "ols-proxy-644444fb76-" is forbidden: 
unable to validate against any security context constraint: 
[provider "anyuid": Forbidden: not  usable by user or serviceaccount, 
provider "pipelines-scc": Forbidden: not usable by user or serviceaccount, 
provider restricted-v2:  .containers[0].runAsUser: Invalid value: 65532: must be in the ranges: [1000840000, 1000849999]
 ---OMITTED---
 ```

Reply:
@bparees Yes, I tried it on `restricted-v2`

But then the container is not created with an error
```text
Error: container create failed: time="2023-12-12T15:32:03Z" level=error msg="runc create failed: unable to start container process: exec: \"uvicorn\": executable file not found in $PATH "
```

To fully support arbitrary users, we will probably have to rewrite the Dockerfile

Reply:
that looks like an error launching the ols container, not the kube-rbac-proxy container.  We 100% control that container and can fix that issue on our side.

Reply:
@vbelouso https://github.com/openshift/lightspeed-service/pull/38 should resolve the issue with running under restricted-v2.  Let me know if you are still having issues.